e2e/features/step_definitions/other-party-subscription.steps.ts

```diff
@@ -0,0 +1,98 @@
+import { Then } from 'cucumber';
+import { expect } from 'chai';
+import { browser } from 'protractor';
+
+const anyCcdPage = new AnyCcdFormPage();
+
+
+Then(/^I subscribed to all parties including other party to "(.+)"$/, async function (isSubscribed) {
+
+ const action = isSubscribed;
+ if (action === 'Yes') {
+    await anyCcdPage.clickElementById('subscriptions_appellantSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_appellantSubscription_tya', 'appellant123')
+    await anyCcdPage.setValueByElementId('subscriptions_appellantSubscription_email', 'appellant-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_appellantSubscription_mobile', '01234567890')
+    await anyCcdPage.clickElementById('subscriptions_appellantSubscription_subscribeEmail_' + action);
+    await anyCcdPage.clickElementById('subscriptions_appellantSubscription_subscribeSms_' + action);
+
+    await anyCcdPage.clickElementById('subscriptions_representativeSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_representativeSubscription_tya', 'representative123')
+    await anyCcdPage.setValueByElementId('subscriptions_representativeSubscription_email', 'representative-test@mailinator.com')
+    await anyCcdPage.setTextFiledValueNull('subscriptions_representativeSubscription_mobile');
+    await anyCcdPage.setValueByElementId('subscriptions_representativeSubscription_mobile', '01234567890')
+
+    await anyCcdPage.clickElementById('subscriptions_appointeeSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_appointeeSubscription_tya', 'appointee123')
+    await anyCcdPage.setValueByElementId('subscriptions_appointeeSubscription_email', 'appointee-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_appointeeSubscription_mobile', '01234567890')
+    await anyCcdPage.clickElementById('subscriptions_jointPartySubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_tya', 'jointParty123')
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_email', 'jointparty-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_jointPartySubscription_mobile', '01234567890')
+
+    await anyCcdPage.clickElementById('subscriptions_supporterSubscription_wantSmsNotifications_' + action);
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_tya', 'supportParty123')
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_email', 'supportparty-test@mailinator.com')
+    await anyCcdPage.setValueByElementId('subscriptions_supporterSubscription_mobile', '01234567890')
+
+     await anyCcdPage.clickElementById('otherParties_0_otherPartySubscription_wantSmsNotifications_' + action);
```

Comment:
the formatting is not correct either on this line or the lines above.

Reply:
Done